odc/stats/plugins/l34_utils/l4_cultivated.py

```diff
@@ -3,93 +3,100 @@
 NODATA = 255
 
 
-def lc_l4_cultivated(l34, level3, lifeform, veg_cover):
+def lc_l4_cultivated(l34, level3, woody, veg_cover):
 
+    woody = expr_eval(
+        "where((a!=a), nodata, a)",
+        {"a": woody.data},
+        name="mask_woody_nodata",
+        dtype="float32",
+        **{"nodata": NODATA},
+    )
+
+    # the 4-8 classes can't happen in LC since cultivated class will not be classified if vegetation doesn't exist.
+    # skip these classes in level4
     l4 = expr_eval(
-        "where((d==110)&(a==111)&(b==10)&(c==1), 9, d)",
-        {"a": level3, "b": veg_cover, "c": lifeform, "d": l34},
+        "where((a==111), 1, d)",
+        {"a": level3, "d": l34},
```

Comment:
there wouldn't be class 1 either, if there is cultivated there must be woody/non

Reply:
Removed this mapping.